docs/water_masking.md

```diff
@@ -0,0 +1,88 @@
+
+This reference dataset is stored in a public AWS S3 bucket: 
+
+`s3://asf-dem-west/WATER_MASK/TILES/`
+
+In the reference raster dataset hosted in AWS, pixels with surface water are assigned a value of 1, and all other pixels are assigned a value of 0. 
+
+***Note that the pixel values used in the reference water mask are opposite to the pixel values used for the water masks included in the InSAR product packages. Refer to the [Applying the Water Mask section](#applying-the-water-mask-during-insar-processing "Jump to Applying the Water Mask during InSAR Processing section") for more information on how the reference water mask is transformed during InSAR processing.***
+
+### Acknowledgments
+
+#### OpenStreetMap
+OpenStreetMap® is open data, licensed under the [Open Data Commons Open Database License](https://opendatacommons.org/licenses/odbl/ "opendatacommons.org/licenses/odbl" ){target=_blank} (ODbL) by the [OpenStreetMap Foundation](https://osmfoundation.org/ "osmfoundation.org" ){target=_blank} (OSMF).
+
+#### ESA WorldCover
+© ESA WorldCover project. Contains modified Copernicus Sentinel data (2020) processed by ESA WorldCover consortium.
+
+## Applying the Water Mask during InSAR Processing
+
+When an InSAR job is submitted for [ASF's On Demand processing](https://storymaps.arcgis.com/stories/68a8a3253900411185ae9eb6bb5283d3 "InSAR On Demand Tutorial" ){target=_blank}, the coordinates of the four corners of the input Sentinel-1 scene are used to find the water mask tile(s) that cover the scene. If the scene crosses multiple tiles, the necessary tiles are mosaicked together. The water mask is then clipped to match the spatial extent of the input Sentinel-1 scene.
+
+The pixel values of the mosaicked and clipped water mask are changed to meet the requirements of the InSAR processing software. **Water pixels are assigned a value of 0, and all remaining pixels are assigned a value of 1.** *Note that these pixel values differ from the reference water mask, where water pixels have a value of 1 and all other pixels have a value of 0.*
+
+If the option to [Apply Water Mask](https://hyp3-docs.asf.alaska.edu/guides/insar_product_guide/#apply-water-mask "InSAR Product Guide - Processing Options - Apply Water Mask" ){target=_blank} was selected by the user submitting the InSAR job, this mask is then used as an input, along with coherence values, to generate the [validity mask](https://hyp3-docs.asf.alaska.edu/guides/insar_product_guide/#masking "InSAR Product Guide - Masking" ){target=_blank} used for phase unwrapping. The 0-value water pixels are excluded from use in phase unwrapping.
+
+A copy of the water mask is always included in the InSAR product package for reference, even if the user chose not to select the option to apply the water mask. In this copy of the water mask, the pixel values are the same as what is used in InSAR processing: pixels indicating water have a value of 0, and all other pixels are assigned a value of 1.
```

Comment:
So why do we store the reference water mask with the opposite pixel values? I may have missed the explanation during one of our sprint reviews.

Reply:
It's a convention thing. Most InSAR tools expect that land is 1 and water is 0.

Reply:
@AndrewPlayer3 can give you more background, but I believe he didn't realize the mask was applied using the 0 values for water (in the hyp3-GAMMA process, the 0 values are applied to the validity mask; pixels with a 0 value for the mask are not included in phase unwrapping). When making the reference mask, true was 1 by default, so he just went with that, not realizing that it was the opposite convention from what's used in InSAR processing.